packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -46,40 +46,55 @@ export function buildRootObject() {
     /** @type WeakStore<ZCFSeat,ZCFSeatAdmin> */
     const seatToZCFSeatAdmin = makeWeakStore('seat');
 
-    const issuers = Object.values(instanceRecord.issuerKeywordRecord);
+    const keywords = Object.keys(instanceRecord.terms.issuers);
+    const issuers = Object.values(instanceRecord.terms.issuers);
 
     const getPromiseForIssuerRecords = issuersP =>
       Promise.all(issuersP.map(issuerTable.getPromiseForIssuerRecord));
 
-    await getPromiseForIssuerRecords(issuers);
-
-    const allSeatStagings = new WeakSet();
-
     const registerIssuerRecord = (keyword, issuerRecord) => {
```

Comment:
Why separate this out from `registerIssuerRecordWKeyword`, which has the error checks, and then call this non-checking one from line 89?

Reply:
Because when I'm overwriting the issuerRecord from Zoe with issuerRecords that include localAmountMath, the keywords will not be unique.